src/index.html

```diff
@@ -8,7 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" type="image/x-icon" href="favicon.ico">
 </head>
-<body class="dark-standard">
+<body class="theme-core">
```

Comment:
Denkt dran das auch in der [Datei in PL](https://github.com/plentymarkets/php-pl/blob/stable7/public/backend/terra.php) zu ändern!

Reply:
Danke, dafür hat @nappynapster bereits nen PR geöffnet: https://github.com/plentymarkets/php-pl/pull/5379/files